frontend/src/metabase/sharing/components/SharingMenu/test/QuestionSharingMenu.unit.spec.tsx

```diff
@@ -199,52 +199,54 @@ describe("QuestionSharingMenu", () => {
 
   describe("embedding", () => {
     describe("admins", () => {
-      it("should show the 'Embed' menu item if embedding is enabled", async () => {
-        setupQuestionSharingMenu({
-          isAdmin: true,
-          isEmbeddingEnabled: true,
+      describe("when embedding is disabled", () => {
+        it("should open the embed modal when the 'Embed' menu item is clicked", async () => {
+          setupQuestionSharingMenu({
+            isAdmin: true,
+            isEmbeddingEnabled: false,
+          });
+          await openMenu();
+          await userEvent.click(screen.getByText("Embed"));
+          expect(await screen.findByText("Embed Metabase")).toBeInTheDocument();
         });
-        await openMenu();
-        expect(screen.getByText("Embed")).toBeInTheDocument();
       });
 
-      it("should show an 'embedding is off' menu item if embedding is disabled", async () => {
-        setupQuestionSharingMenu({
-          isAdmin: true,
-          isEmbeddingEnabled: false,
+      describe("when embedding is enabled", () => {
+        it("should open the embed modal when the 'Embed' menu item is clicked", async () => {
+          setupQuestionSharingMenu({
+            isAdmin: true,
+            isEmbeddingEnabled: true,
+          });
+          await openMenu();
+          await userEvent.click(screen.getByText("Embed"));
+          expect(await screen.findByText("Embed Metabase")).toBeInTheDocument();
         });
-        await openMenu();
-        expect(screen.getByText("Embedding is off")).toBeInTheDocument();
-      });
-
-      it("should open the embed modal when the 'Embed' menu item is clicked", async () => {
```

Comment:
Basically, I copy this test and replace other conditions since now the option will always be clickable.